src/backend/base/langflow/components/helpers/memory.py

```
@@ -197,28 +197,39 @@ async def retrieve_messages(self) -> Data:
+            # For internal memory, we always fetch the last N messages by ordering by DESC
             stored = await aget_messages(
                 sender=sender_type,
                 sender_name=sender_name,
                 session_id=session_id,
-                limit=n_messages,
+                limit=10000,
                 order=order,
             )
-        self.status = stored
+            if n_messages:
+                print("len stored Before", len(stored))
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)